build-tools/packages/build-infrastructure/src/test/fluidRepo.test.ts

```diff
@@ -104,3 +108,52 @@ describe("loadFluidRepo", () => {
 		});
 	});
 });
+
+describe("setDependencyRange", () => {
```

Comment:
Thought of another test case:

setDependencyRange(workspace.packages, releasegroup.packages, depRange)

Assuming the release group is a subset of the workspace, what will happen in this case? Does there need to be a check that the two groups of packages are mutually exclusive?